README.md

```diff
@@ -1,264 +1,505 @@
+
+- [Node `^8.0.0`](https://nodejs.org)
+- [PHP `5.6`, `^7.0.0`](https://php.net)
+- [Composer `^1.0.0`](https://getcomposer.org)
 
-- [Node](https://nodejs.org) v6, v8 is required for pa11y testing
-- PHP 5.4, 5.5, 5.6, OR 7 (5.3 might work, no promises)
-- [`composer`](https://getcomposer.org)
+## Provides
 
-# QuickStart
+- Drupal theme and Pattern Lab app
+- Strict [Atomic Design](http://atomicdesign.bradfrost.com/) component structure
+- Webpack bundling of all CSS, javascript, font, and static image assets for multiple targets (Drupal theme and Pattern Lab)
+- Webpack Dev Server for local hosting and auto asset injection into Pattern Lab and Drupal
+- Auto namespace addition into Drupal theme and Pattern Lab. Within any twig file, `@atoms/thing.twig` means the same thing to Drupal theme and Pattern Lab.
+- Iconfont auto-generation
+- Bootstrap 4 integration, used for all starting example components
+- Auto-linting against the [AirBnB JavaScript Style Guide](https://github.com/airbnb/javascript) and sane Sass standards
+- All Webpack and Gulp files are fully configurable
 
-[Download this repo](https://github.com/phase2/pattern-lab-starter/archive/master.zip), rename folder to `patternlab` and place in themes directory if integrating with Drupal 8, and then `cd` into folder before running below commands.
+## Quickstart
+
+NAMETBD can be run from anywhere to work with Pattern Lab. It also provides a theme to a Drupal website.
+
+### Quickstart anywhere
+
+1. [Download the latest release](https://github.com/phase2/pattern-lab-starter/releases)
+1. Extract anywhere (i.e. this readme should be at `any/where/NAMETBD/README.md`)
+1. Within the extracted folder run:
 
 ```bash
-npm install # or `yarn install` if you want 3x the speed
+npm install
 npm run setup
 npm start
 ```
 
-If you're using Drupal 8, get the [Component Libraries module](https://www.drupal.org/project/components):
+Simply wait until the webpack bundle output appears then visit [http://0.0.0.0/pl](http://0.0.0.0/pl) (or [http://localhost/pl](http://localhost/pl)) and start working.
+
+### Quickstart with Drupal 8
+
+NAMETBD provides a Drupal 8 theme, the starting steps are slightly different:
+
+1. [Download the latest release](https://github.com/phase2/pattern-lab-starter/releases)
+1. Extract to `themes/` at the root of your Drupal 8 install. (i.e. this readme should be at `drupal-root/themes/NAMETBD/README.md`)
+1. Download and install the [Component Libraries module](https://www.drupal.org/project/components):
 
 ```bash
 drush dl components
 drush en components -y
 ```
 
-That's it.
+1. Within `drupal-root/themes/NAMETBD/` run:
+
+```bash
+npm install
+npm run setup
+npm run compile:drupal
+```
+
+This will compile all assets and provide all namespaces to the Drupal theme. Make sure to choose this theme in Drupal Appearance settings and `drush cr` to clear cache.
+
+For subsequent recompile and Drupal cache clear, run:
+
+```bash
+npm run compile:drupal && drush cr
```

Comment:
If I understand correctly, with drush v9 global drush support is going away, this should probably be noted as a path to a Drush install. I've not completed my research into this to understand all the details, so take this as a follow-up caution rather than an immediate action item.

Reply:
Noted.